automation/check-patch.e2e-nmstate-functests.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -xeu
+
+# This script should be able to execute Kubernetes-nmstate
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-nmstate-functests.sh
+
+teardown() {
+    rm -rf "${TMP_COMPONENT_PATH}"
+    make cluster-down
+}
+
+main() {
+    # Setup CNAO and artifacts temp directory
+    source automation/check-patch.setup.sh
+    cd ${TMP_PROJECT_PATH}
+
+    # Spin-up ephemeral cluster with latest CNAO
+    # this script also exports KUBECONFIG, and fetch $COMPONENT repository
+    export KUBEVIRT_NUM_NODES=3 # 1 master, 2 workers
+    export KUBEVIRT_NUM_SECONDARY_NICS=2
+    COMPONENT="nmstate" source automation/components-functests.setup.sh
+
+    trap teardown EXIT
+
+    echo "Configure test parameters"
+    export TIMEOUT=${TIMEOUT:-1h}
+    export NAMESPACE=${NAMESPACE:-cluster-network-addons}
+    export KUBECTL=${KUBECTL:-${TMP_PROJECT_PATH}/cluster/kubectl.sh}
+    export SSH=${SSH:-"${TMP_PROJECT_PATH}/cluster/ssh.sh"}
+    export CLUSTER_PATH=${CLUSTER_PATH:-${TMP_PROJECT_PATH}/_kubevirtci/}
+
+    cat << EOF > ${SSH}
```

Comment:
@qinqon this part rewriting the current content of ssh.sh makes the script get stuck:
```
kubernetes-nmstate (master) $ $SSH
+ node_name=
++ CLUSTER_PATH=/tmp/cnao/cluster-network-addons-operator/_kubevirtci/
++ /tmp/cnao//cluster-network-addons-operator/cluster/kubectl.sh get node --no-headers -o wide
++ awk '{print $6}'
+ node_ip='192.168.66.101
192.168.66.102
192.168.66.103'
+ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null core@192.168.66.101 192.168.66.102 192.168.66.103 --
```

Reply:
removed it, as discussed offline